internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -770,6 +771,7 @@ func (c *ClusterManager) buildResourceSet(cluster api.Cluster) types.ResourceSet
 		c.buildObservabilityCatalogSourceResource(),
 		c.buildObservabilityOperatorGroupResource(),
 		c.buildObservabilitySubscriptionResource(),
+		c.buildDataplaneRhssoCredentialsSecretResource(&cluster),
```

Comment:
I feel like this could be even more concrete. Ok, dataplane uses Red Hat SSO (is that always the case? if ssoservice has not been created with the redhat sso setting that would not be the case), but what is it used for / who uses it?

Reply:
It is used by the Observability Operator to configure the Prometheus to fetch a token and send it along with all remote write requests (more specifically, this section in the Prom config: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#oauth2).

So how about:

`buildRemoteWriteRhssoCredentialsSecret` ?

Reply:
> buildRemoteWriteRhssoCredentialsSecret

Maybe `buildRemoteWriteServiceAccountCredential` as it removes the RH SSO from the method name completely.

Reply:
I think I would associate the name to observability operator

Reply:
Yes, or something similar. 
The important part is that we associate the name to who uses it and what for if it is only for some specific task.
If it is used for multiple tasks for the same element then I would keep out of the name one of the specific tasks.

Reply:
thanks, i've changed the name to `buildObservabilityRemoteWriteServiceAccountCredential`